### PR TITLE
Makes splints fall off when the limb gets shot

### PR DESCRIPTION
**Pull request by MLGTASTICa**
## About The Pull Request

Splints will fall off when the limb they affect is  shot
## Why It's Good For The Game
No longer you can combat pre-splint to avoid a major part of the medical system and balance

## Changelog
:cl:
balance: splints now fall off when the limb they are applied to gets shot.
/:cl:

### CODE REVIEW

**Comment by GrayRachnid** [comment]
This is how CM does it and I thought it happened here too.

**Comment by glaciusinfinity** [comment]
This is a much better nerf than just having them fall off of no reason. 👍

**Comment by FortuneMaster** [comment]
Good practice of ignoring feedback until headcoder tells you that your solution is so shit that there's no way it can be added

**Comment by ValoTheValo** [comment]
How much damage does it take to make a splint fall off? Like a nick from a roach or a amr to the knee?

**Comment by MLGTASTICa** [comment]
> How much damage does it take to make a splint fall off? Like a nick from a roach or a amr to the knee?

any

**Review by TheShown911 — APPROVED** [review]

File: code/modules/mob/living/living_defense.dm [hunk 1]
```diff
@@ -65,6 +65,12 @@
 	//No armor? Damage as usual
 	if(armor_effectiveness == 0)
 		apply_damage(effective_damage, damagetype, def_zone, sharp, edge, used_weapon)
+		if(ishuman(src) && def_zone)
+			var/mob/living/carbon/human/H = src
+			var/obj/item/organ/external/o = H.get_organ(def_zone)
+			if (o && o.status & ORGAN_SPLINTED)
```
Comment (drexample) [inline code/modules/mob/living/living_defense.dm:71]: Any damage and the splints shatter? No deal, add at least some degree of threshold. Slapping someone on the ass and watching their splints fucking annihilate is hilarious, but too stupid.

**Review by drexample — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/living_defense.dm [hunk 2]
```diff
@@ -65,6 +65,12 @@
 	//No armor? Damage as usual
 	if(armor_effectiveness == 0)
 		apply_damage(effective_damage, damagetype, def_zone, sharp, edge, used_weapon)
+		if(ishuman(src) && def_zone)
+			var/mob/living/carbon/human/H = src
+			var/obj/item/organ/external/o = H.get_organ(def_zone)
+			if (o && o.status & ORGAN_SPLINTED && effective_damage > 4)
```
Comment (drexample) [inline code/modules/mob/living/living_defense.dm:71]: Has this been tested? What kind of action wouldn't destroy the splints?

**Pull request closed without merging**